SCClassLibrary/Common/GUI/Base/Grid.sc

```diff
@@ -44,8 +44,9 @@ DrawGrid {
 		y.fontColor = c;
 	}
 	gridColors_ { arg colors;
-		x.gridColor = colors[0];
-		y.gridColor = colors[1];
+		colors = colors.as(Array);
+		x.gridColor = colors.wrapAt(0);
```

Comment:
why `wrapAt`here? Is it to make it forward compatible with an n-dimensional grid?

Reply:
In case only a single color is passed in, both axes get the same color.

Reply:
hm, ok, seems convenient, although in `gridColors = Color.black` the plural s is a little tilted. What about just write `gridColors = Color.black.dup` ?

Reply:
With this implementation, both `gridColors = Color.black` and `gridColors = Color.black.dup` would both work, which seems desirable from a user perspective.
Is the concern that the intention in the method's source is unclear?